Update vsphere.md (#911)
aws/eks-anywhere: docs/content/en/docs/reference/clusterspec/vsphere.md

## VSphereDatacenterConfig Fields

### datacenter (required)

> The vSphere datacenter to deploy the EKS Anywhere cluster on. For example `SDDC-Datacenter`. Note: It's required to add double quotes around the datacenter name in case it has a space in the name of it, example "datacenter one".

Reviewer: ```suggestion
The vSphere datacenter to deploy the EKS Anywhere cluster on. For example `SDDC-Datacenter`. Add double quotes around the datacenter name if there is a space in its name, such as "datacenter one".
```
Small wording suggestions otherwise it looks good to me.
